src/defs/realmRepresentation.ts

```diff
@@ -116,3 +116,21 @@ export default interface RealmRepresentation {
   verifyEmail?: boolean;
   waitIncrementSeconds?: number;
 }
+
+export type PartialImportRealmRepresentation = RealmRepresentation & {
```

Comment:
According to the [documentation](https://www.keycloak.org/docs-api/15.0/rest-api/index.html#_partialimport) this should be named `PartialImportRepresentation`.

```suggestion
export type PartialImportRepresentation = RealmRepresentation & {
```

It also looks to me that [`PartialImportRepresentation`](https://www.keycloak.org/docs-api/15.0/rest-api/index.html#_partialimportrepresentation) doesn't have all of the fields that [`RealmRepresentation`](https://www.keycloak.org/docs-api/15.0/rest-api/index.html#_realmrepresentation) has. Perhaps we should not be extending this existing type, instead adding all the fields explicitly.

Reply:
I don't agree with the shorter name.  When you do the import, the json often contains the entire realm representation that was exported from the Keycloak server at startup.  So even though we don't care about all the fields, it is legal for them to be there.

Reply:
I'd rather follow what the API documentation provides as that is what we do for all other pieces, but if it's really that important I guess we can make an exception.